Microsoft/WebTemplateStudio
# azure page redesign

Changes:
- Added an optional step flag/button that leads you to the last page (improves visibility)
- styling changes for azure services page
- adjusted padding for every page 

How to test:
- make sure the button always routes to the right page 
- should look like this when not logged in: 
![notloggedin](https://user-images.githubusercontent.com/31115095/61001634-d6cddd80-a314-11e9-9047-c4f2979ea4a4.PNG)
- should look like this when logged in:
![loggedin](https://user-images.githubusercontent.com/31115095/61001667-ed743480-a314-11e9-997a-77900f1e736f.PNG)
